protocol/scripts/markets/get_isolated_market_insurance_fund.py

```
@@ -0,0 +1,197 @@
+        command = [binary_path, "q", "module-name-to-address", "insurance_fund:" + str(market_id)]
+        address = run_dydxprotocold(command)
+        market_id_to_address.append((market_id, address))
+
+    return market_id_to_address
+
+def get_bank_balance(market_id_to_address, base_endpoint_url):
+    url = base_endpoint_url + "/cosmos/bank/v1beta1/balances/"
+
+    address_to_balance = {}
+
+    for market_id, address in market_id_to_address:
```

Ruff

Loop control variable `market_id` not used within loop body

Rename unused `market_id` to `_market_id`

(B007)